test_export_gh_config.yaml

```diff
@@ -0,0 +1,51 @@
+# Configuration file for standard GraphHopper routers.
+# 1 profile loaded with default speeds is defined for each vehicle type
+
+graphhopper:
+  datareader.file: ./test-data/micro_nor_cal.osm.pbf
```

Comment:
The only reason we need a separate config here is that graphhopper decides whether or not to create a `CustomGraphHopperOsm` or a `CustomGraphHopperGtfs` based on whether or not the `gtfs.file` field exists in the config. Because we're switching the unit test for the street export to use the nationwide export command, which only reads OSM + creates a `CustomGraphHopperOsm` internally, we need a separate config that doesn't include a gtfs file (tldr: this simulates our nationwide export setup)